.travis.yml

```diff
@@ -7,8 +7,12 @@ php:
   - nightly
   - hhvm
 
-install:
+script: phpunit
+
+before_script:
   - composer self-update
-  - composer install --prefer-source
+  - composer install --dev --optimize-autoloader --prefer-source
```

Comment:
--optimize-autoloader is not really useful here as it's not a production environment. Generating the autoloader will become slower and the optimization gain is not enough to be justified here